frontend/src/components/Brokers/Broker/Configs/TableComponents/InputCell/index.tsx

```diff
@@ -38,6 +33,7 @@ const InputCell: FC<InputCellProps> = ({ row, onUpdate }) => {
 
   const isDynamic = source === 'DYNAMIC_BROKER_CONFIG';
   const configUnit = getConfigUnit(name);
+  const { isReadOnly } = useContext(ClusterContext);
```

Comment:
we shouldn't neglect the `isReadOnly` property of the row, it has a different reason to be disabled -- a broker config property might be r/o